src/sentry/mediators/sentry_apps/creator.py

```diff
@@ -46,7 +47,9 @@ def _create_proxy_user(self):
         return User.objects.create(username=self.name.lower(), is_sentry_app=True)
 
     def _create_api_application(self):
-        return ApiApplication.objects.create(owner_id=self.proxy.id)
+        return ApiApplication.objects.create(
+            owner_id=self.proxy.id, allowed_origins="\n".join(self.allowed_origins)
```

Comment:
Is this list stored in a `text` field or something in the DB?

Reply:
@mnoble Yup...who knows why

Reply:
Can the list of allowed_origins contain newlines in the values?

Reply:
@markstory Yea, I guess so. I just copied a lot of this code from the creation of ApiApplications (see `put` in api_application_details.py) where we do the same sort of new line join. Granted, those are probably barely used so maybe we should be more careful here. Not sure what the impact of having a new line character in a value would be, however.